client/packages/system/src/Patient/Encounter/ListView.tsx

```diff
@@ -9,12 +9,61 @@ import {
   useFormatDateTime,
   useUrlQueryParams,
   ColumnAlign,
+  EncounterNodeStatus,
+  useTranslation,
+  LocaleKey,
+  TypedTFunction,
 } from '@openmsupply-client/common';
 import { EncounterFragment, useEncounter } from './api';
 import { usePatientModalStore } from '../hooks';
 import { PatientModal } from '../PatientView';
 
-type EncounterFragmentWithId = { id: string } & EncounterFragment;
+const effectiveStatus = (
```

Comment:
would be good to lift this out to a utils or helper file: it will be useful in the new encounter list, and would make this file smaller if you could lift it out

Reply:
What do you mean by new encounter list?

Reply:
I will leave it for now and merge...